swatch-producer-azure/src/main/java/com/redhat/swatch/azure/service/BillableUsageAggregateConsumer.java

```diff
@@ -293,7 +296,7 @@ private BillableUsage billableUsageFromAggregateKey(
     billableUsage.setProductId(key.getProductId());
     billableUsage.setMetricId(key.getMetricId());
     billableUsage.setSla(SlaEnum.fromValue(key.getSla()));
-    billableUsage.setSnapshotDate(snapshotDate);
```

Comment:
I was thinking we don't need this anymore but we should keep it so I'll add it back.

Reply:
On second thought I think it would be best to just remove the snapshotDates entirely from the aggregate. We don't need them anymore and it's not worth maintaining it. The only field we end up using now is the UUID when we consume from the dead-letter topic.